src/content/docs/en/guides/client-side-scripts.mdx

```diff
@@ -62,7 +62,20 @@ By default, `<script>` tags are processed by Astro.
 </script>
 ```
 
-To avoid bundling the script, you can use the `is:inline` directive.
+#### Why `type="module"`?
+
+This attribute makes the browser treat the script as a JavaScript module. This has several performance benefits:
+- Rendering is not blocked. The browser continues to process the rest of the HTML while the module script and its dependencies load.
+- The browser waits for HTML to be processed before executing module scripts. You do not need to listen for the "load" event.
+- `async` and `defer` attributes are unnecessary. Module scripts are always deferred.
+
+:::note
+The `async` attribute is valuable for normal scripts because it prevents them from blocking rendering. However, module scripts already have this behavior. Adding `async` to a module script will cause it to execute before the page has fully loaded. This is probably not what you want.
+:::
+
+#### Opting out of processing
```

Comment:
```suggestion
### Opting out of processing
```
Petition to bump this up, as something more important so it gets highlighted in the sidebar TOC!

Reply:
Granted!